backends/qualcomm/scripts/build.sh

```diff
@@ -30,6 +30,14 @@ BUILD_AARCH64="true"
 CMAKE_AARCH64="build_android"
 CLEAN="true"
 
+if(NOT PYTHON_EXECUTABLE)
+  set(PYTHON_EXECUTABLE python3)
+endif()
+
+if(NOT BUCK2)
+  set(BUCK2 buck2)
+endif()
```

Comment:
Bash sematic should be this
```bash
if [ -z PYTHON_EXECUTABLE ]; then
  PYTHON_EXECUTABLE="python3"
fi
 
if [ -z BUCK2 ]; then
  BUCK2="buck2"
fi
```

Reply:
Oh good catch. Update with the comments. What do you think?

Reply:
LTGM. Thanks a lot for helping!